<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ravi-PC" ignored="2" name="ChromeBrowser" tests="2" failures="1" timestamp="2020-05-02T21:49:28 IST" time="9.169" errors="0">
  <testcase name="Login_with_with_set_of_Invalid_data" time="2.225" classname="org.devlab.ui.testbed.LoginTest">
    <failure type="java.lang.AssertionError" message="Failed to login into application with valid set of user :StageUser:StagePwd expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Failed to login into application with valid set of user :StageUser:StagePwd expected [false] but found [true]
at org.devlab.ui.testbed.LoginTest.Login_with_with_set_of_Invalid_data(LoginTest.java:45)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- Login_with_with_set_of_Invalid_data -->
  <testcase name="Login_with_with_set_of_valid_data" time="2.19" classname="org.devlab.ui.testbed.LoginTest"/>
  <testcase name="@BeforeSuite beforeSuiteExc" time="0.0" classname="org.devlab.ui.basetest.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuiteExc -->
  <testcase name="register_user_with_valid_setof_data" time="0.0" classname="org.devlab.ui.testbed.RegistrationTest">
    <ignored/>
  </testcase> <!-- register_user_with_valid_setof_data -->
</testsuite> <!-- ChromeBrowser -->
